Networking/BLIP/BLIPConnection.cc

```diff
@@ -29,6 +29,7 @@
 #include <map>
 #include <unordered_map>
 #include <vector>
+#include "../HTTP/Headers.hh"
```

Comment:
This is inappropriate.  I don't know if it is even needed anymore?  What function does this add?  If it is needed, please correct the CMake project to not need `..` paths like this.

Reply:
For websocket::Headers
In Xcode, "#include Headers.hh" works, but I have to use ",," for Linux and Windows

Reply:
I don't see anywhere that is used in this PR.  If you must have it, then edit the search paths in the CMake project to avoid using parent directory paths.

Reply:
I see where it's used now but it's not something new in this PR.  How was it working before?

Reply:
Before, it used reference &headers, Since I send it to the queue, I changed it by value.